File: Core java/Day02Assignment/src/com/sonata/product.java
package com.sonata;

public class product {
	int prdId;
	String prdName;
	double prdPrice;
	double totalPrice;
	
public product() {
		
	}
	public product(int id,String name,double price) {
		this.prdId = id;
		this.prdName = name;
		this.prdPrice = price;
	}
	
	public void display() {
		System.out.println("Product ID is:" + prdId);
		System.out.println("Product Name is :" + prdName);
		System.out.println("Product Price is:" + prdPrice);
		System.out.println("Product Price with GST is : " + totalPrice);
	}
	
	public double totalPrice() {
		totalPrice = prdPrice + (prdPrice*0.1);
		return totalPrice;
	}
	public static void main(String[] args) {
		product p1 = new product(789, "bat", 2000);
		p1.totalPrice();
		p1.display();
	}

}

File: Typescript assignment and practiced problems/Product.ts
class Product{
    p1:number;
    p2:number;
    p3:number;

    display(){
        if(this.p1>=this.p2 && this.p1>=this.p3){
        console.log(this.p1);}

        if(this.p2>=this.p3 && this.p2>=this.p1){
        console.log(this.p2);}
         
        if(this.p3>=this.p1 && this.p3>=this.p2){
        console.log(this.p3);}
    }
}

var pr1=new Product();
pr1.p1=123;
pr1.p2=125;
pr1.p3=234;
pr1.display();